packages/auth/lib/index.ts

```diff
@@ -1,4 +1,6 @@
-module.exports = {
+import transfromUser = require('./transformUser')
+
+export default {
```

Comment:
I get the following error on staging:
<img width="856" alt="Screenshot 2020-08-29 at 16 56 54" src="https://user-images.githubusercontent.com/410211/91639726-e88eaa80-ea18-11ea-941b-019da4380bf8.png">

Caused by the `revokeConsent` mutation:
https://github.com/orbiting/backends/blob/458cf43e19724854f56b398485475fe8da52786d/packages/auth/graphql/resolvers/_mutations/revokeConsent.js#L1

seems like requiring from this file after changing export style does not work as expected.